sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -62,25 +62,38 @@ public DirectConnectionConfig setConnectionTimeout(Duration connectionTimeout) {
     }
 
     /**
-     * Gets the idle channel timeout
-     * @return idle channel timeout
+     * Gets the idle connection timeout for direct client
+     * Default value is {@link Duration#ZERO}
```

Comment:
Is it to keep the connections warm? 

How are broken connections identified? How will it impact the latencies/performance?

Reply:
Yes, to keep connections warm. When the connections are closed, rntbd logs it. 
We have seen a connection establishment takes anywhere between 200ms - 300ms .